app/controllers/relations_controller.rb

```diff
@@ -0,0 +1,32 @@
+class RelationsController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    @friends = current_user.friends
+  end
+
+  def create
+    @relation = Relation.new(relation_params)
+    if !@relation.save
```

Comment:
Có thể dùng unless, unless == if not
Ngoài ra em nên if else ở đây luôn, tạo được cũng nên có thông báo chứ nhỉ

Reply:
tạo đc thì kết quả đc hiển thị ở view r ạ